src/cfitsio.jl

```diff
@@ -502,12 +553,8 @@ function fits_read_col{T}(f::FITSFile,
                           firstelem::Integer,
                           data::Array{T})
 
-    anynull = Int32[0]
-    status = Int32[0]
-
-    firstrow64 = convert(Int64, firstrow)
-    firstelem64 = convert(Int64, firstelem)
-    nelements64 = convert(Int64, length(data))
```

Comment:
Le 15/03/2015 23:02, Kyle Barbary a écrit :
> In src/cfitsio.jl
> <https://github.com/JuliaAstro/FITSIO.jl/pull/19#discussion_r26452973>:
>
>> @@ -502,12 +553,8 @@ function fits_read_col{T}(f::FITSFile,
>>                            firstelem::Integer,
>>                            data::Array{T})
>>
>> -    anynull = Int32[0]
>> -    status = Int32[0]
>> -
>> -    firstrow64 = convert(Int64, firstrow)
>> -    firstelem64 = convert(Int64, firstelem)
>> -    nelements64 = convert(Int64, length(data))
>
> Should there be a line |nelements = length(data)| here? (|nelements| is
> used below in ccall)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/JuliaAstro/FITSIO.jl/pull/19/files#r26452973>.
>


Yes you are right, I introduced the same error in fits_write_col
these are now fixed